### PR TITLE
Fix/bugs from excel

### PR DESCRIPTION
**Pull request by theketan26**
## Description

Bugs fixes from excel sheet,
1. Shortcut font style and spacing between shortcuts is not according to figma 
2. while uploading any file on formdata then UI is deforming(refer screenshot) 
3. only icon click is navigating user to the the module, whole button should work accordingly
4. Clicking on + button causing UI issue 
5. while clicking or editing any key value pair, unchecked is turning into checked 
6. on a single key-value pair click, select all check box automatically turned on

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by punitkashyup** [comment]
Please attach the issue link

**Comment by punitkashyup** [comment]
Please attach the issue link

**Review by itsmdasifraza — APPROVED** [review]

**Comment by punitkashyup** [comment]
Please attach the issue link

**Comment by theketan26** [comment]
@gc-codes Merge conflicts resolved

File: .github/workflows/approval-check.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  job1:
+    name: check_adminA
+    runs-on: ubuntu-latest
+    env:
+      NAME_ADMIN: adminA
+    steps:
+      - name: st1
+        if: ${{ github.actor != env.NAME_ADMIN }}
+        run: exit 1
+      - name: st2
```
Comment (itsmdasifraza) [inline .github/workflows/approval-check.yml:23]: ??

**Comment by punitkashyup** [comment]
Please attach the issue link

**Comment by punitkashyup** [comment]
Please attach the issue link

**Pull request merged**